Source/GUI/MemViewer/MemViewer.cpp

```diff
@@ -166,7 +216,7 @@ bool MemViewer::handleNaviguationKey(const int key)
     {
       scrollContentsBy(0, 1);
```

Comment:
Here, you have to do `m_byteSelectedPosY = 0;` as you no longer stay the selection to the same place everytime.  Do so after the scroll, this will make the selection jump one row above,